client/lib/analytics/utils/set-tracking-prefs.ts

```diff
@@ -0,0 +1,26 @@
+import cookie from 'cookie';
+import { getTrackingPrefs, TRACKING_PREFS_COOKIE_V2 } from '.';
+import type { TrackingPrefs } from '.';
+
+const COOKIE_MAX_AGE = 15778800; /* six months */
```

Comment:
Sometimes it's more readable (and more precise) to just declare it as a calculation. It makes it easier to understand how we got to that particular number.